Tutorial.ipynb

```diff
@@ -0,0 +1,525 @@
FY6j2+uC7ru83J15CdG0up7mPJ4Bc5+aQ/1ehOPnlNMuFrJzoZ+IsNEYtIb7QxZHJL6UnW3kXCNkrOKU9CqFeQnRfLnH87k5atnER2mQqOQk588PucaFaZizOHG5vRwon+8Uc3v+lOYGs3HdyxkRlYsv/68ngv/tI/b3z7KR+Xd/HhlPl/eu4TUGC2XvryfTZW9rJiSyKqiZMrbR6jqNnCrLy3v7xdYEvAZAbQOjhGmlJMbH8G/gjMmWplMppDJZMdkMtln/9KRQgghhP8K/Lu/5a5RK7W9JpKjtFJn8cnwm3T7MTUthvykSE70iQIGKdFhfHS0G4PFwc/PLUKjlLH+pT3U6w1YnR6KUqMm7FOtlLO6KJl3D3VKnacymYyfrCkgOz6cTZU9JEWHce/qyexqHGR/yzDPXzaNXoONZ7c2AKCLUHPfWQX0jNq46C/7GRlzisT8Wf1pz1cQBDYe6yFLd+o50t0nBilIjmR1kVi3NVpdHGob4fLZGUGEdqLfzJSUKGbljAtXdI1YmeFzEAJo6BP9ewtTxRrxgZZh8pMiSYoSI9cBs51Nlb1cNS9bGi/ZXNnLZ8f1/OrCqWTFhyMIAg9tqKZl0MKPV+bz9s3zmJWtO+OU577mIV7Z3cL6l/YwZHHw8/NKmJQYwSu7WyViS44Oo05vYsBsB8R0bp3ehN23vmnAjMsjSLXuqm6x+Wyar9O6usfIVJ+a1Yl+CwXJwZ93fZ9ZKhGA6BNrcbjJT4qQHsIqOkdZ9MzX1PSIaldFqdH8+Yez2HHfUl6+ehbv3jKfY784i3tWFxAdpuKnG6roNYjv1+3b90s7m4hQKyUZ0cNtI8RHqLh4RrARxOqiZLLjwznaOXpG1/Bk/DMR7T3A6b+NIYQQwv9b8G/9ltuGxCaXtNgwwjUK6YYZiElJkaTFhLGrYYB1v9uNVq2ke9RKpU8C0H/Pv+bvhwlXKylMFfex/qV9KGQylhYkTNgnwKrCZBxuLwcC5m/n58Vjd3mlWuJ1C3NYU5zMq3tamZEZx49W5rOtto/qbiNer8CvP69HgCDBgR11/VR3G095TL8WbmnGxPPUG21sqOjhwunp0mjMgdYhrE4P8/KCHV32NA0RrlYEmcYf6zIwYnVKpFmvN5OlCydSo0QQBOQygq7FIZ8jzzKfUpUgCLy2t40pyVGcXSqmmz8q72ZDRTfLChK5f82UUz4cfBfWlKTw0e0LyYjT8oO/HUImgx+tmMze5iGO+iJwf1rXX5/3SyT6G4bE7ITYCAdipJ4aE0ZRajTDFgepsWHMyYnD6nRjtk/UGfaPfPmREacVZSJ14TQPWPB6BfKTIvF4BZ78rE6qISvkMiYnR7GuNIUZWXFSWvudgx1srhqvx+9pGuIf+9vY0zTEbcvyCFcrsTk9fF6t58LpGRSnRRPme4DMiNPy4hVlpMZocbg8kmpVIAJr2KfCGX0CMpksAzgH+NuZvD6EEEL478R/4rc8ZHFQnBqNUiGnutsoCR8Ewu0RONw2gscr0NBnZtDiYEpKtBSxpflqiNU9Jn783jF+FSDnqDfZeerzBpoHxBThWS9+y299TU1zc3VoVQp2NY47yshkMq5ZkE3TgIWDrSOoFHLOmZbKobYRvqrv56YleTjdXp7d1oBcLuOJ80uClJ60Kjlx4Sr+uKvplOfba7AxJyeOKUmRE9Z90yimU1cHGAPsbhoiUqOUolQ/mgYsTD5pH70GG2kx45q/CrlMSjfbXB52NQ5K0SyIqeRIjZISX6TYoDfROmjhpsW5yGQyjDYX2+v6mJ+nk7SRT4WRMSdfVuvZUtV7yvWzsuO4Y3k+rYNjvHOwkxVFSchl4vH9CFPKUSnF69hntJGlC5fOparLKCor+UQpKjpGSY0JI0yloKrbQHn7KIUp0TT2mRm1ih3GJ78/v3Y0iNrKNb0m8pIiKEiO5ES/iegwFY+fV8zImJMnttSdkuy8XoHHP63h0U01gDir6//on9t2gjXFyVIKfke9KOKxqjgJuVwmReevXTeH6DAVM7PisLm8kl60HzU9Bs75/d7TXms484j2d8BDwGnlemQy2a0ymaxcJpOVu5jY+RVCCCH8V+Cf+i0PDgaPTHi9Al83DDDJRxjtw+Neo4FQK+U43F6pWWfAZKcoJYp6vQlBEGgMqLNtre3j1T2tQdvvbxnmqlcO4fUK9JvsUgNMmErB2VNT0BvFFKAgiOvPL0sjJz6c3SdEAj5naipZunD+9E0LURold63IZ0/TEPubh7hwRjpv3TQXjS9ikctl0vrmgYm1YZPdzZH2UbTqiY0wuxoGSI/VBhGozelhbUlykOzh6JiTIYtD0tr1o9dgCxLXr+kxSjd4/xxwlC+6BXGMyN8IBrDrhCgPuLpYFMT4pnGAr+oHeGDNlFNat7k8Xl76qol73z/GHe9U8OdvWgDoHLZy1zsVUvc3iFHz2VNT2FzVQ4xWxfy8eCmaM9pc2N1eInzXpLHPQteolWzfd6Gq20B+YhQKhRyn20tVt1HSeq7sNCCXQWl6jCQM4o98AawON1PTY0iPHb8u/n/HR4RR1W3ka98DzvllaSwrSOTtgx1c//cjbK/tw2x30TlsZXNVD+f+YS9vHugAYG6OjotmZHDFnEyK06IxO9xcOTdTino/qegmNSZM8pbNiA1jUmKEZBQ/IyuOmVmxfF49/nCyr3mIq149JEXyp8P3Eq1MJjsXGBAE4eh3vU4QhFcEQZgtCMJsFZrvemkIIYTw/wD+ld9yYmKwmH6d3uRrzklgdMyJ0TYxGgGRaJ0eLwmRajJ1Wow2F9MzY8nShdMxMibVyvzYUdcv1XplQKRGyWPnFiGXy1Ap5EENV1FhKg60DCMIAr/f2czCp78GxBTyWwc7cbq9KBVybl2ai0Ypp6JzlKvnZ7OsIEFSIVo4KYFP71oEwJjDw3llaTjcXjZUdE84Fz9fek8RMY053CwrSAyqf26v7SNGG6xGNGC2U5oWTUYAqYKYgvVbA4IYxcplA
```

Comment:
This is the line that can be a constructor parameter instead of being hard-coded.

Reply:
done